pkg/util/utils.go

```diff
@@ -37,6 +40,8 @@ type RetriableError struct {
 	Err error
 }
 
+var VERSION semver.Version = semver.MustParse("0.0.0")
```

Comment:
You might want to make this private, and expose a SetVersion. That way people will have to access it through your function below, which will handle getting/setting it.

It also might make ense to initialize it to nil, to make your "if set yet" check in  GetKubernetesVersion a little easier.